### PR TITLE
fix: various app-builder issues

### PR DESCRIPTION
**Pull request by joshbalfour**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #

### CODE REVIEW

File: packages/app-builder-backend/src/eject/templates/format.ts [hunk 1]
```diff
@@ -9,6 +8,7 @@ export const slugToCamel = (str) =>
 const capitalize = (s) => s && s[0].toUpperCase() + s.slice(1)
 
 export const lint = async (code: string): Promise<string> => {
+  const baseEslint = require('@reapit/ts-scripts/src/eslint/base-eslint') // inline import because ts-scripts has dependency issues
```
Comment (willmcvay) [inline packages/app-builder-backend/src/eject/templates/format.ts:11]: Think this is just because you need to have the eslint executable in scope in your package - can explain in the morning!

File: packages/ts-scripts/src/webpack/webpack.config.dev.js [hunk 2]
```diff
@@ -15,160 +15,161 @@ const generateRegexExcludePackages = () => {
...
-          android: false,
-          appleIcon: false,
-          appleStartup: false,
-          coast: false,
-          firefox: false,
-          opengraph: false,
-          twitter: false,
-          yandex: false,
-          windows: false,
+const webpackConfigDev = ({ appName }) => {
```
Comment (willmcvay) [inline packages/ts-scripts/src/webpack/webpack.config.dev.js:18]: Feels a bit scary because this is used by all the things - bit hard to follow what changed - can you just fill me in in the morning please? Ta!

**Review by willmcvay — CHANGES_REQUESTED** [review]

File: packages/app-builder-backend/src/eject/templates/app.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ export const generateApp = () => {
     import '@reapit/elements/dist/index.css'
 
     const httpLink = createHttpLink({
-      uri: 'https://zbtuirnf0g.execute-api.eu-west-2.amazonaws.com/prod/',
+      uri: 'http://localhost:4000/graphql/',
```
Comment (bashleigh) [inline packages/app-builder-backend/src/eject/templates/app.ts:16]: use an env here instead?

**Review by bashleigh — APPROVED** [review]
Only one small suggestion

File: packages/app-builder-backend/src/eject/templates/app.ts (hunk 3 above)
Comment (joshbalfour) [reply packages/app-builder-backend/src/eject/templates/app.ts]: good shout!

**Review by willmcvay — APPROVED** [review]
LGTM 👍🏻
